gitlab/cli.py

```diff
@@ -194,14 +194,25 @@ def _get_base_parser(add_help: bool = True) -> argparse.ArgumentParser:
         required=False,
         default=os.getenv("GITLAB_URL"),
     )
-    parser.add_argument(
+
+    ssl_verify_group = parser.add_mutually_exclusive_group()
+    ssl_verify_group.add_argument(
         "--ssl-verify",
         help=(
-            "Whether SSL certificates should be validated. [env var: GITLAB_SSL_VERIFY]"
+            "Path to a CA_BUNDLE file or directory with certificates of trusted CAs. "
+            "[env var: GITLAB_SSL_VERIFY]"
         ),
         required=False,
         default=os.getenv("GITLAB_SSL_VERIFY"),
     )
+    ssl_verify_group.add_argument(
+        "--no-ssl-verify",
+        help="Disable SSL verification",
+        required=False,
+        dest="ssl_verify",
+        action="store_false",
+    )
+
```

Comment:
I like the simplicity of this, but it doesn't fully fix #2714. We still cannot use environment variables (this is true for all of our booleans on the CLI). We'd need some kind of strtobool logic for those, and that maybe also falls back to strings for the `ssl_verify` arg. Also, the new flag doesn't have an env var equivalent this way like the others do. WDYT?